protected/routers/cli.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Trait For Routing HTTP Requests
+ */
+trait Router
+{
+    /**
+     * Perform Redirect Rules
+     *
+     * @param string $uri HTTP Request URI
+     */
+    public function routeRedirect(?string $uri = null): void
```

Comment:
Codacy found an issue: [TRUE, FALSE and NULL must be uppercase; expected "NULL" but found "null"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)